### ISSUE TITLE
Write stats db information somewhere for Alex to analyze

### ISSUE THREAD

**Issue by gafferongames** [networknext/next#1471]
Extend relay database to write the stats database values to a binary structure to pubsub, once every minute.

Make this a versioned binary structure. Look at how BillingEntry is done for an example.

Create a pubsub channel and subscriber, "relay_pings" and write the data to this channel. See the pubsub billing code as an example (although you won't need to batch up like it does, the stats db will be large enough).

Next, create a new service called "analytics". Use the billing service as a template for this.

The analytics service should subscribe to this "relay_pings" channel, and when it receives a route matrix, write timestamped entries to a new bigquery table: "relay_pings".

Do not write the *whole* stats db matrix as a structure, instead, write a bigquery entry for each relay pair in the triangular route matrix. Grab me (glenn), when you create this bigquery table, and I'll show you how to do it, and how to design and partition it on timestamp.

Basically, each entry should probably be:

timestamp timestamp
relay_a int
relay_b int
rtt float
jitter float
packetLoss float

This will unblock Alex to do analysis on relay stats, which is probably the #1 thing we can do to improve the quality of our relay fleet!
